content/history/raceday.inc

```diff
@@ -143,15 +190,17 @@
   <li class="nav-item">
     <a id="mens-tab" href="#tab-mens" class="nav-link active" data-toggle="tab" role="tab" aria-controls="tab-mens" aria-selected="true">Men's Results</a>
   </li>
-  <li class="nav-item">
-    <a id="womens-tab" href="#tab-womens" class="nav-link" data-toggle="tab" role="tab" aria-controls="tab-womens" aria-selected="false">Women's Results</a>
-  </li>
+  <?php if ($year >= 1979) { ?>
+    <li class="nav-item">
+      <a id="womens-tab" href="#tab-womens" class="nav-link" data-toggle="tab" role="tab" aria-controls="tab-womens" aria-selected="false">Women's Results</a>
+    </li>
+  <?php } ?>
   <?php if(count($pheatsArr)>0) { ?>
     <li class="nav-item">
       <a id="prelimheats-tab" href="#tab-prelimheats" class="nav-link" data-toggle="tab" role="tab" aria-controls="tab-prelimheats" aria-selected="false">Prelim Heats</a>
     </li>
   <?php } ?>
-  <?php if(count($fheatsArr)>0) { ?>
+  <?php if($year >= 1979 && count($fheatsArr)>0) { ?>
```

Comment:
Why does this one have the year check?

Reply:
Similar to the other spot in this file -- we don't display either women's tab before 1979, since there will never be data for them.  So, if we display the tab, it results in a click target that never has anything.

I will check to see if these can both work by just checking for an empty array instead (I think this one can, I'm less sure about the first) and if so, just check for empty arrays rather than hard coding the year.

Reply:
Gross, this is totally a mistake on my part.

But it also made me realize the earlier year check actually can just be a check for the array having content, which is just cleaner.